pkg/managed/clusterroles.go

```diff
@@ -211,9 +211,14 @@ func newClusterRoles(binding *v1beta1v8o.VerrazzanoBinding, clusterName string)
 			},
 			{
 				APIGroups: []string{"rbac.authorization.k8s.io"},
-				Resources: []string{"clusterroles", "clusterrolebindings", "roles", "rolebindings"},
+				Resources: []string{"clusterroles", "roles"},
 				Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
 			},
+			{
+				APIGroups: []string{"rbac.authorization.k8s.io"},
+				Resources: []string{"clusterrolebindings", "rolebindings"},
+				Verbs:     []string{"get", "list", "watch", "create", "update", "delete", "patch"},
```

Comment:
It is not clear why the patch verb needed?  Does this mean the verrazzano operator needs to patch a clusterrolebinding or rolebinding?

Reply:
The weblogic-kubernetes-operator needed the patch permission.  Here is the error from the wko log for the clusterrolebinding.  

cannot patch \"verrazzano-bobs-books-binding-weblogic-operator-clusterrolebinding-general\" with kind ClusterRoleBinding: clusterrolebindings.rbac.authorization.k8s.io \"verrazzano-bobs-books-binding-weblogic-operator-clusterrolebinding-general\" is forbidden: User \"system:serviceaccount:verrazzano-system:verrazzano-system

There was a similar error for the rolebinding.